source/Irrlicht/CBawFile.h

```diff
@@ -45,6 +60,7 @@ namespace irr { namespace core
 			EBT_COUNT
 		};
 
+		//! Meant to be used as pointer to blob's data
 		uint8_t data[1];
```

Comment:
Maybe boot this and replace with a 'void* getData() {return this;}'

Reply:
Yea, better